lab/lab1/part3-data-transformation.html

```diff
@@ -62,16 +62,43 @@
     if their formatting is especially wrong. But it should accommodate most points.
 
     ===================== */
-
-    var data = bikeArrayClean;
+    var raw = bikeArrayDirty;
+
+    let data = [];
+
+    let cleaning = (obj) => {
+      console.log(obj);
+      temp = [];
+      if(obj[0].includes("-7")) {
+        n = 0
+      } else {n=1};
+      temp.push(obj[n], obj[n+1]);
+      if(typeof obj[n+2] === 'string') {
+        if(!obj[n+2].includes('stations')) {
```

Comment:
The solution I pointed you earlier --- https://github.com/MUSA611-CPLN692-spring2019/cpln692-week3/blob/5c110ed9e162e3619164438282440657650ae232/lab/lab1/part3-data-transformation.html#L76 basically does a loop in a loop to check every single entry. And checks to see if they are numbers doing something similar to what you've done. You have a pretty good solution here - if you really want additional feedback we can try to call @moradology  (this is in reference to @bucklerd s comment on the pull request if there is a better way to do this bike data cleaning.